packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -0,0 +1,144 @@
+import { css } from "@microsoft/fast-element";
+import {
+    disabledCursor,
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+import {
+    accentFillActiveBehavior,
+    accentFillHoverBehavior,
+    accentFillSelectedBehavior,
+    accentForegroundCutRestBehavior,
+    neutralFillHoverBehavior,
+    neutralFillStealthHoverBehavior,
+    neutralFillStealthRestBehavior,
+    neutralFillStealthSelectedBehavior,
+    neutralFocusBehavior,
+    neutralFocusInnerAccentBehavior,
+    neutralForegroundHoverBehavior,
+    neutralForegroundRestBehavior,
+    neutralLayerL1Behavior,
+} from "../styles/recipes";
+import { heightNumber } from "../styles/size";
+
+export const OptionStyles = css`
+    ${display("inline-flex")} :host {
+        border-radius: calc(var(--corner-radius) * 1px);
+        border: calc(var(--focus-outline-width) * 1px) solid transparent;
+        box-sizing: border-box;
+        color: ${neutralForegroundRestBehavior.var};
+        cursor: pointer;
+        fill: currentcolor;
+        font-size: var(--type-ramp-base-font-size);
+        height: calc(${heightNumber} * 1px);
+        line-height: var(--type-ramp-base-line-height);
+        margin: 0 calc(var(--design-unit) * 1px);
+        outline: none;
+        overflow: hidden;
+        padding: calc(var(--design-unit) * 2.25px);
+        user-select: none;
+        white-space: nowrap;
+    }
+
+    :host(:${focusVisible}) {
+        box-shadow: 0 0 0 calc(var(--focus-outline-width) * 1px) inset ${
+            neutralFocusInnerAccentBehavior.var
+        };
+        border-color: ${neutralFocusBehavior.var};
+        background: ${accentFillHoverBehavior.var};
+        color: ${accentForegroundCutRestBehavior.var};
+    }
+
+    :host(.selected) {
```

Comment:
There was a previous PR that moved many of these selectors away from using a class on the host, to using the attr selector. That is because classes on the host can get clobbered in some frameworks. Can this use an attribute selector?

Reply:
For `option`s, he `selected` attribute represents the `defaultSelected` property. The `selected`-ness of the option is only represented by its `selected` property, which is only expressed in the DOM via the `selected` class and the `aria-selected` attribute.

It looks like `checkbox` is using `[aria-checked="true"]` for styling, so I'll go that route and use `[aria-selected="true"]`.